### ISSUE TITLE
Opening MM appears to set 'default wallet' to off

### ISSUE THREAD

**Issue by mr-michael** [tallycash/extension#668]
tldr I install Tally and the toggle works fine. After opening MM at some point, I have default turned on I am no longer to connect to dapps I was previously connecting to (even if I'm displayed the 'injected' option for UNI or the 'tally' option for Curve).

It simply loads forever and then fails.

Then I go back to the Tally menu and default wallet is turned off. I turn it on then go to another tab. Then go back to Menu and it's turned off. I turn it on. Then exit and re-enter the Menu. It's turned off again.

What it looks like when I try to connect:
<img width="950" alt="Screen Shot 2021-12-15 at 1 19 40 PM" src="https://user-images.githubusercontent.com/7005061/146244343-d23d63ec-cd2e-4ddb-9dd0-ab8e198272fe.png">
<img width="1151" alt="Screen Shot 2021-12-15 at 12 45 00 PM" src="https://user-images.githubusercontent.com/7005061/146244345-da69538b-a0b9-4964-b9f8-4ecd28bb6c59.png">
<img width="1207" alt="Screen Shot 2021-12-15 at 12 45 48 PM" src="https://user-images.githubusercontent.com/7005061/146244346-1799ea4d-ab95-477b-b2c1-d07321f7ef97.png">

**Comment by mr-michael** [comment]
Seems related? https://github.com/tallycash/extension/issues/660

**Comment by greg-nagy** [comment]
This is fixed with: https://github.com/tallycash/extension/pull/620

Not closing so we can double check when the PR has landed.

**Comment by greg-nagy** [comment]
actually closing it. let's reopen if it is not working
